### DISCUSSION TITLE
SQLite Studio (paid version) on Microsoft Store: is it official?

### DISCUSSION THREAD

**Discussion by Gary92** [pawelsalawa/sqlitestudio#4968] [General]
I just found an app called "SQLite Studio" on the Microsoft Store, which (based on the screenshots) looks identical to the one available here but it is paid. Therefore, I wanted to ask: is there an "official" release of the software on the Microsoft Store too (many software are available from both the proprietary site and the store), or did I simply find the work/expedient of an independent dev?

**Comment by pawelsalawa** [comment]
It's not official. It looks like someone created the MS Store package and if it's a legit package - then it's okay. Anybody is allowed to use or even sell GPL-licensed code (just like Linux has commercial distributions), as long as it's properly attributed - and here I see small misuse from the author of that package, because he does not attribute neither me or project's homepage. At the same time, the application itself has the "About" menu option, which kind of fulfills the attribution requirement (assuming that package author didn't alter the code and didn't remove attributions there before building the package).

Long story short, it looks to be simply a packaged distribution of SQLiteStudio to be available through MS Store. Same as the one available here for free.
